server/users.go

```diff
@@ -50,8 +58,7 @@ func (r *userRequest) ValidRoles() error {
 	if len(r.Roles) > 0 {
 		for _, r := range r.Roles {
 			switch r.Name {
-			// TODO: add SuperAdmin
-			case ViewerRoleName, EditorRoleName, AdminRoleName:
+			case MemberRoleName, ViewerRoleName, EditorRoleName, AdminRoleName:
 				continue
 			default:
 				return fmt.Errorf("Unknown role %s. Valid roles are 'viewer', 'editor', 'admin', and 'superadmin'", r.Name)
```

Comment:
Update to have 'member' (or guest or whatever)

Reply:
Opened https://github.com/influxdata/chronograf/issues/2212